src/tiler/package.json

```diff
@@ -4,8 +4,9 @@
   "description": "",
   "main": "server.js",
   "dependencies": {
-    "windshaft": "0.36.0",
-    "aws-sdk": "2.1.45"
+    "aws-sdk": "2.1.45",
+    "rollbar": "^0.5.12",
```

Comment:
After looking it up, `^` appears to allow changes to `MINOR` version, whereas `~` only allows updates to `PATCH`. I think we want to only allow flexibility with `PATCH`.